messaging/src/main/java/org/axonframework/queryhandling/QueryGateway.java

```diff
@@ -240,6 +238,7 @@
      * @see QueryBus#subscriptionQuery(SubscriptionQueryMessage)
      * @see QueryBus#subscriptionQuery(SubscriptionQueryMessage, SubscriptionQueryBackpressure, int)
      */
+    @Deprecated
```

Comment:
The JavaDoc should reference why it's deprecated. In that line of documentation, it should refer to the method/approach which should be used instead of this function.